### DISCUSSION TITLE
Get dependence non decision

### DISCUSSION THREAD

**Discussion by TinusUU** [mwshinn/PyDDM#115] [Ask for help] [answered]
Hi,

For pyDDM 0.8.1 we can now define non-decision time as: nondecision = "tr" instead of using overlay.
However, m.get_dependence("nondecision") does not work. 
How can I get the dependence for nondecision? 

(Since for startingpoint m.get_dependence("IC") still holds, I tried using "overlay" or "OverlayNonDecision"  instead, but this doesn't work either).

Thanks,
Martijn

**Comment by mwshinn** [comment, marked as answer]
You can use `m.get_dependence("overlay")` but you said this does not work, could you please provide more details on what you mean by "does not work"?

Also, what is the reason for trying to access the object?  You should normally not need this.  If you want the parameter value, you can use `m.parameters()`.

**Reply by TinusUU** [reply]
Ah yes, you are right. It does work! (I received an error, but this was due to the parameter that I didn't pass through the lambda function - my bad). I use it to get learning-rate in a loss function (RLDDM, to test for learning effects in nondecision time)
Thanks!
